port.h

```diff
@@ -116,7 +116,15 @@ SOFTWARE.
 #define M_PI 3.14159265359
 #endif
 
+#endif  // CURRENT_WINDOWS
+
+#ifdef CURRENT_APPLE
+#ifdef CURRENT_APPLE_ENABLE_NSLOG
+#define CURRENT_NSLOG(...) ::NSLog(__VA_ARGS__)
+#else
+void CURRENT_NSLOG(...) {}
```

Comment:
Why `{}`? I don't know Objective C well, but won't an empty define do it?

If the users may put a semicolon after `CURRENT_NSLOG(...)`, perhaps some `static_cast<void>(0)` is a safer call?